metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -336,6 +342,16 @@ class UnityCatalogSourceConfig(
         description="Generate usage statistics.",
     )
 
+    usage_data_source: UsageDataSource = pydantic.Field(
+        default=UsageDataSource.AUTO,
```

Comment:
Previously, only option was rest api, right?
And now default is auto (that is system tables with fallback to rest api).

May that be considered a breaking change?
Existing recipes may start to fail because of _eg_ missing privileges on the `system.query.history` system table?

Regarding the breaking change: I would add a note in `updating_datahub` or keep the default as API, so there is no breaking change

Regarding permissions: do we need to document new permissions for `system.query.history` system table?

Reply:
Yeah, ideally, we should use SQL wherever possible. Good point, I will add a note to the breaking change section of the doc.